src/Illuminate/Support/functions.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Illuminate\Support\Process\PhpExecutableFinder;
```

Comment:
```suggestion
use Symfony\Component\Process\ExecutableFinder;
use Illuminate\Support\Process\PhpExecutableFinder;
```

You forgot to add the `ExecutableFinder`. Otherwise this throws a "Class "Illuminate\Support\Process\ExecutableFinder" not found" error